app/assets/javascripts/veterans.js.coffee

```diff
@@ -9,6 +9,7 @@ $ ->
   osmUrl = '//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
   osmAttrib = 'Map data © <a href="http://openstreetmap.org">'
   osmAttrib += 'OpenStreetMap</a> contributors'
+  markers = L.markerClusterGroup()
```

Comment:
@jjhampton Great point! What do you think of this? It's at maxClusterRadius:25.
<img width="721" alt="screen shot 2017-03-18 at 4 02 43 pm" src="https://cloud.githubusercontent.com/assets/13701741/24075608/2b10191c-0bf5-11e7-8cd2-d51ee4c7f867.png">